guides/user/pipeline/triggers-with-artifactsrewrite/webhooks/index.md

```diff
@@ -44,8 +44,9 @@ endpoint depends on how you've configured your [Spinnaker
 endpoints](/setup/security) -- if you're running on a different endpoint, for
 example `https://api.spinnaker-prod.net`, that'll be shown instead.
 
-Keeping track of that endpoint as `$ENDPOINT` (it will depend on where
-Spinnaker is installed), save that pipeline, and run:
+Keeping track of that endpoint as `$ENDPOINT` will depend on where
```

Comment:
Maybe just:
"Save that pipeline and run:
[command, including $ENDPOINT]
Where $ENDPOINT is the URL for the endpoint, which depends on where Spinnaker is installed."

Or something like that?

Reply:
These would work better as imperative verbs in numbered steps, but you'd have to do that to the entire article.
This paragraph here is the third step in this section.